app/AcquireIntersonArrayRF.cxx

```diff
@@ -121,18 +121,18 @@ int main( int argc, char * argv[] )
   ContainerType container;
 
   const int height = hwControls.GetLinesPerArray();
+  std::cout << "Lines per array: " << height << std::endl;
   const int width = ContainerType::MAX_RFSAMPLES;
+  std::cout << "Max RF samples: " << width << std::endl;
   const itk::SizeValueType framesToCollect = frames;
+  std::cout << "Version info: " << hwControls.ReadFPGAVersion() << std::endl;
```

Comment:
It would be good to make a comment here that the ReadFPGAVersion() is necessary and not optional for probe initialization..